e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/A_PublishSingleNodeOrchestratedTestTheory.cs

```diff
@@ -303,26 +321,47 @@ public class PublishSingleNodeOrchestratedTestTheory
             request.AddHeader(TestConstants.HttpHeaderNames.Authorization, accessToken);
             request.Resource = string.Format(TestConstants.APIRoutes.PublisherJobsFormat, _context.OpcUaEndpointId);
 
-            var response = await client.ExecuteAsync(request, cts.Token);
+            var response = client.ExecuteAsync(request, cts.Token).GetAwaiter().GetResult();
             Assert.NotNull(response);
-            Assert.True(response.IsSuccessful, "DELETE /publisher/v2/jobs/{jobId} failed!");
 
             if (!response.IsSuccessful) {
                 _output.WriteLine($"StatusCode: {response.StatusCode}");
                 _output.WriteLine($"ErrorMessage: {response.ErrorMessage}");
+                Assert.True(response.IsSuccessful, "DELETE /publisher/v2/jobs/{jobId} failed!");
             }
         }
 
         [Fact, PriorityOrder(12)]
-        public async Task Test_VerifyNoDataIncomingAtIoTHub() {
+        public void Test_VerifyNoDataIncomingAtIoTHub() {
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-            await Task.Delay(90 * 1000, cts.Token); //wait till the publishing has stopped
+            Task.Delay(90 * 1000, cts.Token).GetAwaiter().GetResult(); //wait till the publishing has stopped
```

Comment:
>.GetAwaiter().GetResult() [](start = 44, length = 25)

this is really ugly, there must be a good explanation why async can not be used...

Reply:
see https://github.com/Azure/Industrial-IoT/pull/1020#discussion_r563553027